app/javascript/controllers/file_auth_controller.js

```diff
@@ -133,6 +134,11 @@ export default class extends Controller {
                                        // with a link to the media server file location (and media token)
                                        // and this can happen with a non-media object that happens to have
                                        // a media file in it, e.g. ds777pr3860
+        
+        if(this.isLocationRestricted(json, contentResourceId)) {
```

Comment:
I thought we were going to do location restriction completely on the backend, so that we were able to show the location restriction banner even when they were in the location.  with that decision, I don't know why we want/need any location-restricted stuff in the frontend.